app/tests/test_operations.cpp

```diff
@@ -49,21 +49,38 @@ TEST_CASE("edge_collapsing", "[test_operation]")
 
     fastEnvelope::FastEnvelope envelope;
     TetWild tetwild(params, envelope);
-
-    std::vector<VertexAttributes> vertices(4);
-    vertices[0].m_posf = Vector3f(0, 0, 0);
-    vertices[1].m_posf = Vector3f(1, 0, 0);
-    vertices[2].m_posf = Vector3f(0, 1, 0);
-    vertices[3].m_posf = Vector3f(0, 0, 1);
-    std::vector<std::array<size_t, 4>> tets = {{{0, 1, 3, 2}}};
-    std::vector<TetAttributes> tet_attrs(1);
-
-    tetwild.init(vertices.size(), tets);
-    tetwild.create_mesh_attributes(vertices, tet_attrs);
+    {
```

Comment:
Strange, these are from other pr.

Reply:
Do not copy file from other branches. Merge instead: it creates a lot of conflicts this way.

Reply:
I did not. I merged from some branches Teseo or Daniele worked on previous